ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -868,7 +869,11 @@ gst_tensor_src_tizensensor_start (GstBaseSrc * src)
 
   /* 3. Fire it up! */
   self->running = TRUE;
-  sensor_listener_start (self->listener);
+  if (sensor_listener_start (self->listener) != 0) {
+    self->running = FALSE;
+    retval = FALSE;
+    goto exit;
```

Comment:
Although the current implementation of the function can handle already configured sources, you can consider cleaning up the set configuration in case of failure.

Reply:
I think `_ts_clean_up_handle (self)` could work for this case... Am I right?

Reply:
yes.